stdlib/task/src/main/java/org/ballerinalang/stdlib/task/utils/Utils.java

```diff
@@ -69,11 +68,9 @@ public static void setError(Context context, String message) {
         context.setReturnValues(createError(context, message));
     }
 
-    public static String getCronExpressionFromAppointmentRecord(BValue record)
-            throws SchedulingException {
-
+    public static String getCronExpressionFromAppointmentRecord(BValue record) throws SchedulingException {
         String cronExpression;
-        if (record instanceof BMap && RECORD_APPOINTMENT_DATA.equals(record.getType().getName())) {
+        if (record instanceof BMap) {
```

Comment:
What you need to check here is whether this is a record right? Ideally you should be getting the type of `record` and checking its type tag to see if it's a record. BMap is used for JSON and objects as well.

Reply:
Done.